src/pineko/evolve.py

```diff
@@ -116,16 +116,16 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     xif = 1.0 if sv_method is not None else tcard["XIF"]
     operators_card["configs"]["scvar_method"] = sv_method
     q2_grid = (xif * xif * muf2_grid).tolist()
-    atlas = ThresholdsAtlas(
-        masses=np.array([tcard["mc"], tcard["mb"], tcard["mt"]]) ** 2,
-        q2_ref=tcard["Q0"] ** 2,
-        nf_ref=tcard["nfref"],
-        thresholds_ratios=np.array([tcard["kcThr"], tcard["kbThr"], tcard["ktThr"]])
-        ** 2,
-        max_nf=tcard["MaxNfPdf"],
+    masses = np.array([tcard["mc"], tcard["mb"], tcard["mt"]]) ** 2
+    thresholds_ratios = np.array([tcard["kcThr"], tcard["kbThr"], tcard["ktThr"]]) ** 2
+    for q in range(tcard["MaxNfPdf"], 6 + 1):
```

Comment:
```suggestion
    for q in range(tcard["MaxNfPdf"] + 1, 6 + 1):
```
I think here we need to loop only thresholds  higher than the last heavy quark included no?

Reply:
Yes I agree